src/sql/workbench/contrib/notebook/browser/cellViews/textCell.component.ts

```diff
@@ -102,6 +105,9 @@ export class TextCellComponent extends CellView implements OnInit, OnChanges {
 				this.markdownResult.dispose();
 			}
 		}));
+		this._register(this._configurationService.onDidChangeConfiguration(e => {
```

Comment:
You still want to init the value to the current config when creating the text cell since this will only fire when a user actually changes the config

Reply:
Added the config value to ngOnInit